docs/sources/enterprise/query-caching.md

```diff
@@ -22,3 +22,7 @@ Query caching currently works for all backend data sources. You can enable the c
 ## Enable query caching
 
 To enable and configure query caching, please refer the the [Query caching section of Enterprise Configuration]({{< relref "./enterprise-configuration.md#query-caching" >}}).
+
+## Skip caching per request
+
+Requests that contain the header `X-Cache-Skip` with any value will be skipped. This can be particularly useful when debugging datasource queries using cURL.
```

Comment:
```suggestion
Requests that contain the header `X-Cache-Skip` with any value are skipped. This can be particularly useful when debugging data source queries using cURL.
```

Reply:
By "skipped" do you mean they are not cached? If so, then we should say that. Also, if you can state this in an active way, then the sentence would be clearer.

Reply:
Correct.

:+1: I'll update that